### PR TITLE
Error messages for login form

### PR DESCRIPTION
**Pull request by sahilsharma011**
#152

### CODE REVIEW

File: resources/views/auth/login.blade.php
Comment (kapilsharma) [inline resources/views/auth/login.blade.php]: Indentation wrong in two lines.

Only these changes are correct for issue #153. Other changes are coming due to merge on other issue. If we merge this PR, all other PR will be merged automatically. Thus, before we can merge this PE, we need to merge other PRs and update this branch from master.

**Review by kapilsharma — CHANGES_REQUESTED** [review]
Code from other issues also present in this PR. Thus it can be merged only once other PRs are merged and updated in this branch.

**Comment by sahilsharma011** [comment]
@kapilsharma I've made the changes you asked for

File: composer.json
Comment (kapilsharma) [inline composer.json]: These changes are not related to login form.

This probably happened due to wrong branch creation/merges.

File: database/seeds/August16ArticleSeeder.php
Comment (kapilsharma) [inline database/seeds/August16ArticleSeeder.php]: That is separate PR. Why it is showing as part of this PR?

These changes are not related to login form.

This probably happened due to wrong branch creation/merges.

File: database/seeds/DatabaseSeeder.php
Comment (kapilsharma) [inline database/seeds/DatabaseSeeder.php]: These changes are not related to login form.

This probably happened due to wrong branch creation/merges.

File: database/seeds/MagazineTableSeeder.php
Comment (kapilsharma) [inline database/seeds/MagazineTableSeeder.php]: These changes are not related to login form.

This probably happened due to wrong branch creation/merges.

File: resources/views/PhpReboot/Magazine/articleDetails.blade.php
Comment (kapilsharma) [inline resources/views/PhpReboot/Magazine/articleDetails.blade.php]: These changes are not related to login form.

This probably happened due to wrong branch creation/merges.

File: resources/views/PhpReboot/Magazine/monthPage.blade.php
Comment (kapilsharma) [inline resources/views/PhpReboot/Magazine/monthPage.blade.php]: These changes are not related to login form.

This probably happened due to wrong branch creation/merges.

File: resources/views/auth/login.blade.php
Comment (kapilsharma) [inline resources/views/auth/login.blade.php]: Only these 9 lines should be part of new PR.

Can you please make new PR with only required code.

**Review by kapilsharma — CHANGES_REQUESTED** [review]
This PR have lot of code which is not part of the issue. Probably happened due to incorrect branching/merging.

@sahilsharma011 in case of any doubt, lets have slack chat or Skype call on git branching for forking + git flow workflow.

**Pull request closed without merging**
